src/pages/Quiz/QuizPage.jsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from "react";
+import PageLayout from "../../components/PageLayout";
+import Button from "@mui/material/Button";
+import LinearProgress from '@mui/material/LinearProgress';
+import { v4 as uuidv4 } from "uuid";
+import { Box } from "@mui/system";
+
+/**
+ * Renders the Assessment page.
+ * @returns {JSX.Element} QuizPage component
+ */
+
+function QuizPage() {
+
+    const [currentQuestion, setcurrentQuestion] = useState(0);
+    const [progressValue, setprogressValue] = useState(1);
```

Comment:
Why is the default value for the progress Bar 1?

Reply:
It was the value I had in there when it finally started working. It also sets a minimal amount of progress even before answering the first question and I thought that looked cuter :)

Reply:
Why does it not work with 0?

Reply:
it does work with zero, I just think it looks better with one